doc/cprover-manual/function-contracts.md

```diff
@@ -0,0 +1,502 @@
+# Function Contracts
+
+Every developer has the idea of an implementation in their heads. This is an
```

Comment:
Comment: might be helpful to start from a proof harness and transform it into a function contract (either adding this as a section or starting from that as an example)

Reply:
We will add the comparison with proof harnesses in our https://github.com/awslabs/aws-templates-for-cbmc-proofs/wiki.